platform/app/public/config/default.js

```diff
@@ -7,6 +7,9 @@ window.config = {
   showStudyList: true,
   // some windows systems have issues with more than 3 web workers
   maxNumberOfWebWorkers: 3,
+  investigationalUseDialog: {
```

Comment:
please add this to documentation in https://docs.ohif.org/configuration/configurationFiles 

also with the `groupEnabledModesFirst` 

also we need to make this default for all the configs

Reply:
actually make that a default, then provide options to change, so config files should not change at all